ninjapanzer/get_out_and_about
# Adding Devise to resolve #3

Resolves #3 
### QA
- Bundle and migrate
  - `bundle install`
  - `rake db:migrate`
- Start the application and `mailcatcher` as described in the `README.md`
- Create a new user
- Confirm the user
- Sign in
